package.json

```diff
@@ -52,12 +52,10 @@
     "chai": "^4.2.0",
     "eslint": "^7.6.0",
     "eslint-config-nebenan": "^3.0.0",
-    "mocha": "^8.2.1",
-    "react": "^17.0.1",
+    "mocha": "^10.2.0",
     "react-router": "^5.2.0"
   },
   "peerDependencies": {
-    "react": "^16.13.1 || ^17.0.1",
```

Comment:
but why dont we keep the peerDependency? is that already in react-router?

Reply:
We removed that peerDependency!

Reply:
Or am I misunderstanding you here?

Reply:
yes, why did you remove it?

Reply:
it's a react package after all

Reply:
Aaaah I messed up peerDependencies and devDependencies ☕️ 
Sorry for the confusion, fixed it, good catch!